colorbleed/maya/lib.py

```diff
@@ -1354,9 +1353,20 @@ def validate_fps():
 
     """
 
+    fps = lib.get_project_fps()  # can be int or float
     current_fps = mel.eval('currentTimeUnitToFPS()')  # returns float
-    fps = lib.get_project_fps()
-    if fps != current_fps:
+
+    # Check if FPS is supported in Maya and compare with current fps
+    if isinstance(fps, int) and fps in INT_FPS:
```

Comment:
The INT_FPS checks, etc. are redundant - should be removed.

It should just check whether `current_fps == project_fps`

```python
fps = lib.get_project_fps()  # can be int or float
current_fps = mel.eval('currentTimeUnitToFPS()')  # returns float

if current_fps != fps:
    # show warning
```